invenio_administration/assets/semantic-ui/js/invenio_administration/actions/DeleteModal.js

```diff
@@ -55,34 +56,40 @@ export class DeleteModal extends Component {
     const { loading, error } = this.state;
     const { modalOpen, toggleModal, children, title } = this.props;
     return (
-      <Modal role="dialog" open={modalOpen}>
-        <Modal.Header as="h2">
-          <Trans defaults="Delete {{title}}" values={{ title: title }} />
-        </Modal.Header>
-        <Modal.Content>
-          <Modal.Description>
-            {children}
-            {!isEmpty(error) && <ErrorMessage {...error} />}
-          </Modal.Description>
-        </Modal.Content>
-        <Modal.Actions>
-          <Button
-            icon="cancel"
-            onClick={() => {
-              this.cleanError();
-              toggleModal(false);
-            }}
-            loading={loading}
-            content={i18next.t("Cancel")}
-            floated="left"
-            size="medium"
-          />
-          <Button negative onClick={this.handleOnButtonClick} loading={loading}>
-            <Icon name="trash alternate" />
-            {i18next.t("Delete")}
-          </Button>
-        </Modal.Actions>
-      </Modal>
+      <Overridable id="DeleteModal" {...this.props}>
+        <Modal role="dialog" open={modalOpen}>
+          <Modal.Header as="h2">
+            <Trans defaults="Delete {{title}}" values={{ title: title }} />
+          </Modal.Header>
+          <Modal.Content>
+            <Modal.Description>
+              {children}
+              {!isEmpty(error) && <ErrorMessage {...error} />}
+            </Modal.Description>
+          </Modal.Content>
+          <Modal.Actions>
+            <Button
+              icon="cancel"
+              onClick={() => {
+                this.cleanError();
+                toggleModal(false);
+              }}
+              loading={loading}
+              content={i18next.t("Cancel")}
+              floated="left"
+              size="medium"
+            />
+            <Button
+              negative
+              onClick={this.handleOnButtonClick}
+              loading={loading}
+            >
+              <Icon name="trash alternate" />
+              {i18next.t("Delete")}
+            </Button>
+          </Modal.Actions>
+        </Modal>
+      </Overridable>
     );
   }
 }
```

Comment:
add `export default Overridable("DeleteModal", DeleteModal)` where you override the whole component